src/unix/aix.c

```diff
@@ -946,12 +954,17 @@ char** uv_setup_args(int argc, char** argv) {
 int uv_set_process_title(const char* title) {
   char* new_title;
 
+  uv_once(&once, init_once);
+  uv_mutex_lock(&mutex);
```

Comment:
You can move the `uv_mutex_lock()` call to right before the `if (process_title_ptr != NULL)` check.